File: Unit 1 Fundamentals of programming/Fundamentals Topic 2 Selection/Topic 2 Python Programs/W2 Q6 car rental.py
#AS AQA unit 1 W2 Q7 car rental
print("Enter a choice of:")
print("    1 Economy car")
print("    2 Saloon car")
print("    3 Sports car")
selection1 = int(input("Choice: "))
choice = "valid"
if selection1 == 1:
    print("You chose Economy car")
elif selection1 == 2:
    print("You chose Saloon car")
elif selection1 == 3:
    print("You chose Sports car")
else:
    print("Invalid choice")
    choice = "invalid"
 

if choice=="valid":
    selection2 = input("Do you wish to proceed (P) or cancel (C)? ")
    selection2 = selection2.upper()
    selection2FirstLetter = selection2[0]
    if selection2FirstLetter == "P":
        print("You chose to proceed")
    elif selection2FirstLetter =="C":
        print("You chose to cancel")
    else:
        print("Invalid entry")
    print("Have a nice day")
input("\nPresss Enter to exit")
           

File: Unit 1 Fundamentals of programming/Fundamentals Topic 2 Selection/Topic 2 VB Programs/W2 Q6 Car Rental.vb
Module Module1

    Sub Main()
        'W2 Q4 Car Rental
        Dim selection As Integer = 0
        Dim selection2 As String = ""
        Console.WriteLine("Enter a choice of:")
        Console.WriteLine("1 Economy Car")
        Console.WriteLine("2 Saloon Car")
        Console.WriteLine("3 Sports Car")
        selection = Console.ReadLine()

        'Use of a Select Case as an alternative to IF..THEN..ELSEIF..
        Select Case selection
            Case 1
                Console.WriteLine("You chose Economy Car")
            Case 2
                Console.WriteLine("You chose Saloon Car")
            Case 3
                Console.WriteLine("You chose Sports Car")
            Case Else
                Console.WriteLine("Invalid choice")
        End Select


        Console.WriteLine("Do you wish to proceed or cancel?")

        selection2 = Console.ReadLine()

        'Using .ToLower converts the answer to lower case to avoid case errors
        If selection2.ToLower = "proceed" Then
            Console.WriteLine("You chose to proceed")
        ElseIf selection2.ToLower = "cancel" Then
            Console.WriteLine("You chose to cancel")
        Else : Console.WriteLine("Invalid entry")
        End If
        Console.WriteLine("Have a nice day")


    End Sub

End Module
